File: 99_CS50/pset1/mario/mario.c
//get libraries
#include <stdio.h>
#include <cs50.h>


//start script
int main(void)

{
    //introduce user entered variable
    int height;


    //prompt user for height and validate (1 < height < 8)
    do
    {
        height = get_int("Height: ");
    }
    while (height > 8 || height < 1);


    //draw right adjusted pyramid with a gap and then a left adjusted pyramid
    //add in new row
    for (int i = 1; i <= height; i++)
    {
        //add in spaces
        for (int n = height - i; n > 0; n--)
        {
            printf(" ");
        }

        //add in left side hashes
        for (int j = 1; j <= i; j++)
        {
            printf("#");
        }
        
        //add in gap
        printf("  ");
        
        //add in right side hashes
        for (int j = 1; j <= i; j++)
        {
            printf("#");
        }
        
        printf("\n");
    }
}

File: 99_CS50/pset6/mario/less/mario.py
# get pyramid height, ensure valid input.
height = 0
while height < 1 or height > 8:
    try:
        height = int(input('Height: '))
    except ValueError:
        continue

# print spaces and hashes
for i in range(1, height + 1):
    j = height - i
    print(" " * j, end='')
    print("#" * i)